nautobot/core/templates/inc/footer.html

```diff
@@ -2,53 +2,103 @@
 {% load helpers %}
 {% load plugins %}
 
-<style>
-    /* Styles for viewports narrower than 1232px, i.e. bootstrap 'xs' and 'sm' layouts */
-    @media (max-width: 1232px) {
-        .footer-col-1, .footer-col-2, .footer-col-3 {
-            text-align: center !important;
-        }
-    }
-    @media (max-width: 767px) {
-        .footer .footer-links {
-            white-space: normal !important;
-        }
-    }
```

Comment:
I presume the new layout still gracefully handles small browser windows? I'll try to pull it down and test it out later today.

Reply:
Looks fine to me in Firefox at least. :-)